numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2434,39 +2467,51 @@ def launch_kernel(
     args,
     cooperative=False,
 ):
-    param_ptrs = [addressof(arg) for arg in args]
-    params = (c_void_p * len(param_ptrs))(*param_ptrs)
-
-    params_for_launch = addressof(params)
-    extra = 0
-
-    if cooperative:
-        driver.cuLaunchCooperativeKernel(
-            cufunc_handle,
-            gx,
-            gy,
-            gz,
-            bx,
-            by,
-            bz,
-            sharedmem,
-            hstream,
-            params_for_launch,
-        )
+    # Convert stream handle to cuda.core Stream object
```

Comment:
This seems odd because we just unpacked it at the call site, ex:
https://github.com/NVIDIA/numba-cuda/blob/9212523b2e09f587e151a496bd1cf87f234f56c9/numba_cuda/numba/cuda/dispatcher.py#L478

Reply:
Removed.